launchable/commands/subset.py

```diff
@@ -416,7 +418,7 @@ def rel_base_path(path):
 
         def stdin(self) -> Union[TextIO, List]:
             # To avoid the cli continue to wait from stdin
-            if is_get_tests_from_previous_sessions:
+            if self.is_get_tests_from_previous_sessions:
                 return []
```

Comment:
That `stdin()` method is returning `Union[TextIO,List]` feels like a leaky abstraction, though granted, it was not introduced in this PR. I guess the backward compatibility ties our hands, but even then, there are ways to evolve these things. I'll see if I can create a follow-up PR.